models/question.js

```diff
@@ -39,6 +39,17 @@ module.exports = function(sequelize, DataTypes) {
                     },
                     as: "createdByUser"
                 });
+
+                /*
+                 * Ready for usage in later development
+                 *  
+                Question.belongsToMany(models.Tag, {
```

Comment:
Mislim da je bolje `Question.hasMany` u ovom slučaju, jer to više semantički odgovara.
Takođe, ako ovako prošireni modeli rade, i prave odgovarajuće kolone i tabele u bazi, odkomentariši kod.
Repozitorijum ne bi trebao da sadrži komentarisan kod.